manifests/mets.py

```diff
@@ -163,22 +163,23 @@ def process_intermediate(div, instance_var, new_ranges=None):
                 if is_page(sd):
                         my_range = process_page(sd, instance_var)
                 else:
-			#logger.debug("process_intermediate: processing int div: " + div.get('LABEL') )
+                        #logger.debug("process_intermediate: processing int div: " + div.get('LABEL') )
                         my_range = process_intermediate(sd, instance_var)
-			#logger.debug("process_intermediate: my_range: " + str(my_range) )
+                        #logger.debug("process_intermediate: my_range: " + str(my_range) )
                 if my_range:
-			#logger.debug("process_intermediate: appending ranges for int div: " + div.get('LABEL') + " range: " + str(my_range) )
+                        #logger.debug("process_intermediate: appending ranges for int div: " + div.get('LABEL') + " range: " + str(my_range) )
                         new_ranges.append(my_range)
 
         # this is for the books where every single page is labeled (like Book of Hours)
         # most books do not do this
-        if len(new_ranges) == 1:
-		#logger.debug("process_intermediate: returning a new_range w/ len 1")
-                return {get_rangeKey(div): new_ranges[0].values()[0]}
+		#### 1/6/2021 cg- this breaks nested labels! - problematic
+        #if len(new_ranges) == 1:
```

Comment:
It appears that this `if` block was introduced to resolve an earlier issue: https://github.com/harvard-library/metadata_server/commit/bc67fb21c002956a3400161d61fd6234da9df1cd

Are we certain that by removing the block now we are not reintroducing the previous error?

Reply:
Talked with Chip re: this, putting the conclusion of said convo here so you can see it - the issue here was with specific patterns in object TOCs - specifically, a number of records were created with one intermediate per-page whose labels replicated or functionally replicated the page labels. 

At the time, it was decided that collapsing legitimate cases of one-intermediate-per-page structure was preferable to having the illegitimate cases balloon out to double their size with a layer of nesting.  The _ideal_ solution is to remediate objects with that structure in their description by removing/rewriting the intermediates, and removing this code.  In any event, removing this code won't _break_ anything, it'll just render things with the one-intermediate-per-page structure more irritating (a real problem, but not something that will block access).

Reply:
Thanks, @pobocks and @cgoines, for the background and rationale. This should be good to go.